hack/update-codegen.sh

```diff
@@ -45,15 +45,10 @@ fi
 
 export GOMODCACHE GO111MODULE GOFLAGS GOPATH
 
-# Even when modules are enabled, the code-generator tools always write to
-# a traditional GOPATH directory, so fake on up to point to the current
-# workspace.
-mkdir -p "$GOPATH/src/sigs.k8s.io"
-ln -sf "${SCRIPT_ROOT}" "$GOPATH/src/sigs.k8s.io/network-policy-api"
-
+readonly API_VERSION=v1alpha1
```

Comment:
I like that we now have a single place in the script for version. Let's maybe also add a reminder that everytime there's a spec bump this need to be updated